packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,17 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map or as these directories:\n  ` +
+                searchQueue.join(`\n  `)
```

Comment:
probably still a good idea to explain how to bust caches since watchman and the RN packager cache implementations are buggy afaik